build.sbt

```diff
@@ -20,13 +54,17 @@ lazy val api = (project in file("api"))
   )
 
 lazy val templates = (project in file("templates"))
-  .settings(Common.settings: _*)
+  .settings(settings: _*)
   .dependsOn(
     common
   )
 
 lazy val `integration-test` = (project in file("integration-test"))
+  .configs(ItTest)
```

Comment:
This didn't work for me, I ran `make test` (which is just `sbt test`) and it ran the integration tests. Do you see the same?

Reply:
You're right, I forgot one setting property for `integraton-test` build